docs-src/src/layouts/Navigation.js

```diff
@@ -43,280 +83,10 @@ export default function Navigation() {
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/CloseButton/">Close Button</a>
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/Icon/">Icon</a>
-          </Menu.Item>
-        </Menu>
-        <Menu vertical>
-          <Menu.Item text>
-            <h5>Utilities</h5>
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/AuthenticityToken/">Authenticity Token</a>
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/BrowserSupportWarning/">
-              Browser Support Warning
-            </a>
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/HelpText/">Help Text</a>
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/Loader/">Loader</a>
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/Progress/">Progress</a>
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/Social/">Social</a>
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/TextAlign/">Text Align</a>
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/Visibility/">Visibility</a>
-          </Menu.Item>
-        </Menu>
-        <Menu vertical>
-          <Menu.Item text>
-            <h5>Prototyping</h5>
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/Lipsum/">Lorem Ipsum</a>
-          </Menu.Item>
-        </Menu>
-        <Menu vertical>
-          <Menu.Item text>
-            <h5>Map</h5>
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/Map/DefaultMap/">Default Map Component</a>
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/Map/HybridMap/">Hybrid Map</a>
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/Map/DesignedSilverMap/">
-              Small Designed Silver Map
-            </a>
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/Map/DesignedRetroMap/">Designed Retro Map</a>
-          </Menu.Item>
-        </Menu>
-        <Menu vertical>
-          <Menu.Item text>
-            <h5>Layout Examples</h5>
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/LayoutExamples/LogIn/">Log In</a>
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/LayoutExamples/Registration/">Registration</a>
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/LayoutExamples/Landing/">Landing Page Sections</a>
-          </Menu.Item>
-          <Menu.Item>
-            <a href="/components/LayoutExamples/Pricing/">Pricing Section</a>
-          </Menu.Item>
-        </Menu>
-      </nav>
+        {/* Search bar */}
+        <Input className='search' placeholder='Search for Component' value={formValue} onChange={handleFormChange}/>
```

Comment:
This search function seems super useful, especially if it could be reused in other projects and not just for this navigation. Try making it "dumber" and importing it into the Navigation as a separate "Search Bar" component that would be able to accept an array of any type rather than one solely intended for this navigation menu.

Reply:
Tagging along here, and adding some general feedback from our design jamming session today. Please see ref the screenshot below.

<img width="250" alt="Screen Shot 2021-01-27 at 3 28 47 PM" src="https://user-images.githubusercontent.com/49654454/106061967-b793ce00-60bb-11eb-887e-73621a766d65.png">